mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -857,6 +857,19 @@ func resourceSqlDatabaseInstanceCreate(d *schema.ResourceData, meta interface{})
     }
     <% end -%>
 
+	patchCalls := []func(opts ...googleapi.CallOption) (*sqladmin.Operation, error){}
```

Comment:
If we were to have more fields in the future that needed to use Patch in postcreate, why wouldn't we just add them to the same `&sqladmin.DatabaseInstance` and make a single patch call?

Reply:
good point ! pushed a change to remedy that  :)